service/README.md

```diff
@@ -48,6 +48,22 @@ You should get a response similar to this:
   {"status":"UP","diskSpace":{"status":"UP","total":249644974080,"free":137188298752,"threshold":10485760}}
 ```
 
+Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/import-ccd-definition.sh
+ to http://fpl-service:4000 - this ensures the CCD use the docker networking to reach
+the service.
+
+### Run the application (from IntelliiJ)
+
+For local profile - configure the notify.api_key if necessary (in service/src/main/resources/application.yaml). 
```

Comment:
I would not encourage people to make changes there, they will surely commit it one day. I'd rather say they need to override value via settings of IntelliJ runner.

Reply:
Sure that's much better! I will push a change for it.